{"trial_id": "0029", "hyperparameters": {"space": [{"class_name": "Boolean", "config": {"name": "use_second_lstm", "default": false, "conditions": []}}, {"class_name": "Boolean", "config": {"name": "use_attention_1", "default": false, "conditions": []}}, {"class_name": "Boolean", "config": {"name": "use_attention_2", "default": false, "conditions": []}}, {"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 16, "max_value": 640, "step": 24, "sampling": "linear"}}, {"class_name": "Boolean", "config": {"name": "bidir_1", "default": false, "conditions": []}}, {"class_name": "Int", "config": {"name": "units_f", "default": null, "conditions": [], "min_value": 16, "max_value": 640, "step": 24, "sampling": "linear"}}, {"class_name": "Boolean", "config": {"name": "bidir_f", "default": false, "conditions": []}}, {"class_name": "Boolean", "config": {"name": "dense_2", "default": false, "conditions": []}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 1e-05, "conditions": [], "min_value": 1e-05, "max_value": 0.01, "step": null, "sampling": "log"}}, {"class_name": "Int", "config": {"name": "reduce_lr_patience", "default": null, "conditions": [], "min_value": 2, "max_value": 5, "step": 1, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "reduce_lr_factor", "default": 0.4, "conditions": [], "values": [0.4, 0.5, 0.6, 0.7], "ordered": true}}, {"class_name": "Int", "config": {"name": "attn_units_1", "default": null, "conditions": [], "min_value": 92, "max_value": 368, "step": 24, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_dense_2", "default": null, "conditions": [], "min_value": 93, "max_value": 558, "step": 24, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 16, "max_value": 640, "step": 24, "sampling": "linear"}}, {"class_name": "Boolean", "config": {"name": "bidir_2", "default": false, "conditions": []}}, {"class_name": "Int", "config": {"name": "attn_units_2", "default": null, "conditions": [], "min_value": 90, "max_value": 270, "step": 24, "sampling": "linear"}}], "values": {"use_second_lstm": false, "use_attention_1": false, "use_attention_2": false, "units_1": 472, "bidir_1": false, "units_f": 88, "bidir_f": false, "dense_2": true, "learning_rate": 0.002620095734527658, "reduce_lr_patience": 3, "reduce_lr_factor": 0.5, "attn_units_1": 92, "units_dense_2": 189, "units_2": 16, "bidir_2": true, "attn_units_2": 186, "tuner/epochs": 3, "tuner/initial_epoch": 0, "tuner/bracket": 5, "tuner/round": 0}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/tuners/hyperband.py\", line 427, in run_trial\n    return super().run_trial(trial, *fit_args, **fit_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n    results = self.hypermodel.fit(hp, model, *args, **kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n    return model.fit(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/utils/traceback_utils.py\", line 122, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/tensorflow/python/eager/execute.py\", line 59, in quick_execute\n    except TypeError as e:\ntensorflow.python.framework.errors_impl.InternalError: Graph execution error:\n\nDetected at node gradient_tape/past_future_model_1/lstm_1/CudnnRNNBackpropV3 defined at (most recent call last):\n  File \"/home/jr/Desktop/TFG_Pronostico_secuencias_temporales/3_lstm/lstm_futuro.py\", line 242, in <module>\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 234, in search\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/tuners/hyperband.py\", line 427, in run_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/utils/traceback_utils.py\", line 117, in error_handler\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 371, in fit\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 219, in function\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 132, in multi_step_on_iterator\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 113, in one_step_on_data\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 77, in train_step\n\nFailed to call DoRnnBackward with model config: [rnn_mode, rnn_input_mode, rnn_direction_mode]: 2, 0, 0 , [num_layers, input_size, num_units, dir_count, max_seq_length, batch_size, cell_num_units]: [1, 5, 472, 1, 30, 64, 472] \n\t [[{{node gradient_tape/past_future_model_1/lstm_1/CudnnRNNBackpropV3}}]] [Op:__inference_multi_step_on_iterator_565109]\n"}